upt_macports/tests/test_macports_backend.py

```diff
@@ -64,6 +64,25 @@ def test_version_conversion(self):
             self.assertEqual(
                 self.macports_backend.cpan_version(mp_ver), cpan_ver)
 
+    @mock.patch('upt.Backend.needs_requirement')
+    def test_require(self, mock_need_req):
```

Comment:
can we give this a bit clearer name, for example somthing like ```test_needs_requirements```?

Reply:
Done.